hibernate-core/src/test/java/org/hibernate/orm/test/query/hql/InsertConflictOnConstraintTest.java

```diff
@@ -42,6 +44,7 @@ public class InsertConflictOnConstraintTest {
 	@RequiresDialect( MySQLDialect.class )
 	@RequiresDialect( HSQLDialect.class )
 	@RequiresDialect( DerbyDialect.class )
+	@SkipForDialect( dialectClass = MariaDBDialect.class )
```

Comment:
Can you please add a reason for the skip?

Reply:
after correction (add the missing inheritance of MySQLDialect buildSQLExceptionConversionDelegate method), this is now ok